presto-parser/src/main/java/com/facebook/presto/sql/tree/RollbackToSavepoint.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.tree;
+
+import java.util.Objects;
+import java.util.Optional;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static java.util.Objects.requireNonNull;
+
+public final class RollbackToSavepoint
```

Comment:
They're logically two distinct commands and have separate `DataDefinitionTask`. The task needs to be different so that `getName()` is correct, and if combined, the logic would be harder.

This is similar to `CreateTable` and `CreateTableAsSelect`. Just pretend the SQL is actually something different like `RESTORE SAVEPOINT`.